packages/patternlab/styleguide/source/_patterns/02-molecules/callout-link.twig

```diff
@@ -1,30 +1,50 @@
+{% set calloutLinkTemplate = calloutLink.download ? " ma__callout-link--download" : "" %}
 
-{# <div > #}
-  <a href="{{ calloutLink.href }}"
-  class="ma__callout-link {{calloutLinkTheme}}"
-  {% if alloutLink.info %} title="{{ calloutLink.info }}"{% endif %}
-  target="">
-    {% if calloutLink.eyebrow or calloutLink.time %}
-    <div class="ma__callout-link__header">
-      {% if calloutLink.eyebrow %}
-        <span class="ma__callout-link__eyebrow">{{ calloutLink.eyebrow }}</span>
-      {% endif %}
-      {% if calloutLink.time %}
-        <span class="ma__callout-link__time">{{ calloutLink.time }}</span>
-      {% endif %}
-    </div>
-   {% endif %}
-      {% if calloutLink.text %}
-    	<span class="ma__callout-link__container">
-    		<span class="ma__callout-link__text" >{{ calloutLink.text }}&nbsp;{{ icon('arrow') }}</span>
-    	</span>
-      {% endif %}
-    {% if calloutLink.emphasized %}
-      <span class="ma__callout-link__emphasized">{{ calloutLink.emphasized }}</span>
+
+<a href="{{ calloutLink.href }}"
+class="ma__callout-link {{calloutLinkTheme}} {{calloutLinkTemplate}}"
+{% if calloutLink.info %} title="{{ calloutLink.info }}"{% endif %}
+target="">
+  {% if calloutLink.eyebrow or calloutLink.time %}
+  <div class="ma__callout-link__header">
+    {% if calloutLink.eyebrow %}
+      <span class="ma__callout-link__eyebrow">{{ calloutLink.eyebrow }}</span>
     {% endif %}
-    {% if calloutLink.description %}
-      <p class="ma__callout-link__description">{{ calloutLink.description }}</p>
+    {% if calloutLink.time %}
+      <span class="ma__callout-link__time">{{ calloutLink.time }}</span>
     {% endif %}
-  </a>
-{# </div> #}
+  </div>
+  {% endif %}
+    {% if calloutLink.text %}
+      {% if not calloutLink.download %}
+    <span class="ma__callout-link__container">
+      <span class="ma__callout-link__text" >{{ calloutLink.text }}&nbsp;{{ icon('arrow') }}</span>
+    </span>
+      {% else %}
+          {% set sizeClass = calloutLink.iconSize ? "ma__download-link--" ~ calloutLink.iconSize : "" %}
+          <div class="ma__download-link ma__callout-link__container {{ sizeClass }}">
+          {% if calloutLink.icon %}
+              <div class="ma__download-link__icon">
+                  {{ icon(calloutLink.icon) }}
+              </div>
+          {% endif %}
+          <span class="ma__download-link__file-link ma__callout-link__text">
+                  <span class="visually-hidden">Open {{ calloutLink.format }} file{% if calloutLink.size %}, {{ calloutLink.size }},{% endif %} for</span>
+                  {{ calloutLink.text }}
+              </span>
+          {% if calloutLink.format or calloutLink.size %}
+              <span class="ma__download-link__file-spec">({{ calloutLink.format }} {{ calloutLink.size }})</span>
+          {% endif %}
+              <span class="ma__callout-link__text">{{ icon('arrow') }}</span>
```

Comment:
```suggestion
```
I think we can remove this trailing icon, to keep it consistent with download link